packages/esm-patient-test-results-app/src/hiv/concepts.ts

```diff
@@ -0,0 +1,58 @@
+const concepts = {
+  display: 'Bloodwork',
+  concept: 'ConvSet', // convienence set
+  obs: [],
+  subSets: [
+    {
+      display: 'Hematology',
+      concept: 'ConvSet',
+      obs: [],
+      subSets: [
+        {
+          display: 'Complete Blood Count',
+          concept: 'LabSet',
```

Comment:
Since you already have the `display`, are you using `concept: 'LabSet'` for anything?

Reply:
Currently there is no use for this on the frontend. However it could provide some additional information if we later decide to extend the feature. I will remove this for now.

Reply:
I generally tend to avoid the temptation of including/implementing what is not currently needed. I postpone it to the future when the times comes for it. You will be shocked at many unused features in software, which were added with the hope that they may be needed in future, but they never ever get needed. 😊

Reply:
Sounds good